templates/rhsm-subscriptions-capacity-ingress.yml

```diff
@@ -55,6 +55,8 @@ parameters:
     value: '500'
   - name: DEVTEST_SUBSCRIPTION_EDITING_ENABLED
     value: 'false'
+  - name: DEVTEST_EVENT_EDITING_ENABLED
+    value: 'false'
```

Comment:
Needs a corresponding section in the pod environment variables:
```yaml
                - name: DEVTEST_EVENT_EDITING_ENABLED
                  value: ${DEVTEST_EVENT_EDITING_ENABLED}
```

Reply:
Added DEVTEST_EVENT_EDITING_ENABLED in pod environment variables.Thank you.